server/server.js

```diff
@@ -269,10 +270,18 @@ app.get('/genotet', function(req, res) {
 
     // Bed data queries
     case 'bed':
-      var file = req.query.file;
+      var file = req.query.fileName;
       var chr = req.query.chr;
-      var dir = bedPath + file + '_chr/' + file + '_' + chr;
-      data = bed.readBed(dir, req.query.xl, req.query.xr);
+      var dir = bedPath + file + '_chr/' + file + '_chr' + chr;
```

Comment:
I got the bed file from Emily's email whose file name is Tcell_max4_FDRp15.bed. Assume that the 'file' is Tcell_max4_FDRp15 (I think it should be bedName, but it seems that the bedName and fileName mapping part is not finished yet?). I understand that you are going to only read those lines whose first word is the chromosome name (chr). But I do not understand this path name. Could you please give me some explanation? Thank you very much!